apps/chat/src/utils/app/application.ts

```diff
@@ -143,17 +165,15 @@ export const convertApplicationFromApi = (
     type: EntityType.Application,
     id,
     inputAttachmentTypes: application.input_attachment_types,
+    applicationProperties: application.application_properties,
```

Comment:
`document_relative_url` should be decoded
```suggestion
    applicationProperties: mapApplicationPropertiesFromApi(
      application.application_properties,
    ),
```